frontend/packages/client/src/components/Card/styles.js

```diff
@@ -0,0 +1,37 @@
+const Card = {
+  baseStyle: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: 6,
+    border: '1px',
+    borderColor: 'black',
+  },
+  variants: {
+    warning: {
+      alignItems: 'flex-start',
+      padding: 6,
+      borderRadius: '2xl',
+      borderColor: 'red.500',
+      backgroundColor: 'red.1000',
+    },
+    flowBox: {
+      gap: 1,
+      paddingTop: '14px',
+      paddingBottom: '14px',
+      paddingLeft: '16px',
+      paddingRight: '16px',
+      display: 'flex',
+      flexDirection: 'column',
+      borderRadius: 'lg',
+      backgroundColor: 'white',
+      borderColor: 'grey.200',
+      minWidth: '210px',
+      height: '69px',
+    },
+  },
+  defaultProps: {
+    background: 'white',
+  },
+};
+
+export default Card;
```

Comment:
this is a custom component, therefore defining the style here makes it easier to identify it as a custom component and not mixing this definition with the theme overwrites.